### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jennahyasin**

### CODE REVIEW

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java
Comment (YukioRivera) [inline cst438_register/src/main/java/com/cst438/controller/StudentController.java:1]: The code appears well-structured and organized. The required user stories of adding a student, adding a hold, and deleting a hold have been implemented proficiently. It is noteworthy that the StudentDTO function has been appropriately utilized across the controller, which adds to the clarity and coherence of the code.

File: cst438_register/src/test/java/com/cst438/JunitTestStudent.java
Comment (YukioRivera) [inline cst438_register/src/test/java/com/cst438/JunitTestStudent.java:1]: The code has a clear structure and is easy to read and understand.
The use of mocks in the tests makes them fast and reliable, allowing for thorough testing of the code.
The test cases are comprehensive and cover all important scenarios.
The use of the StudentDTO class to return student data is a good design choice, as it separates concerns and makes the code more modular.
The use of HTTP status codes in the responses is a good practice and makes it easier to handle errors and exceptions.

**Review by YukioRivera — COMMENTED** [review]
Overall this is a good project and looks like it would function well. It covers all the stories and the Junit tests test all the proper commands. Good job!

**Comment by jennahyasin** [comment]
Thanks for the comments! I will add some documentation and merge to the master branch.

**Pull request merged**